projects/batfish/src/main/antlr4/org/batfish/grammar/cisco/Cisco_line.g4

```diff
@@ -71,6 +71,7 @@ l_null
       | FLUSH_AT_ACTIVATION
       | HISTORY
       | IPV6
+      | LENGTH DEC
```

Comment:
This will conflict with #392

Reply:
Thanks, Dustin! I'll make sure we handle your PR first.